src/st/np3/4E69C.c

```diff
@@ -167,7 +167,71 @@ void func_801CF778(void) {
     et->unk9C = et->next->ext.et_801CF254.unk9C + 0x300;
 }
 
-INCLUDE_ASM("asm/us/st/np3/nonmatchings/4E69C", func_801CF7A0);
+int func_801CF7A0(s16* arg0) {
```

Comment:
Looking at this scratch, it seems like arg0 is an Entity* https://decomp.me/scratch/51iIJ

Reply:
Thanks, implemented along with some further deductions! Incidentally it seems like the bogus return value might be an original bug, because who needs compiler warnings.